### ISSUE TITLE
Warn user if they attempt to quit with unsaved changes

### ISSUE THREAD

**Issue by audionerd** [wonderunit/storyboarder#449]
```
when user tries to quit
  if they have unsaved changes (e.g. is board file marked dirty)
    and autosave is OFF
      ask "you have unsaved changes. sure to quit?"
        YES
          quit
        NO
          cancel
          user must manually save, then manually attempt to close again, and it'll work
    if autosave is ON
      save
      quit
```

**Comment by audionerd** [comment]
Related to #221

**Comment by audionerd** [comment]
Fixed in 8ea3c37.
